package.json

```diff
@@ -96,6 +96,7 @@
     "react": "^15.3.1",
     "react-addons-transition-group": "^15.3.0",
     "react-addons-update": "^15.3.1",
+    "react-copy-to-clipboard": "^5.0.0",
```

Comment:
I don't think it would a problem to do the copy button without an NPM package. 
Do you guys think I should keep it or remove it completely?